### ISSUE TITLE
Editor button

### ISSUE THREAD

**Issue by jeegg** [sandmanmedia/sandman-control#1]
I love this theme but I have three issues.

1. When installing on Joomla! 3.4.3 I got an error that there's no en-GB.tpl_sandmanctrl.ini in the package, though it installed fine. I have tried adding another file with such name, the error was gone during install then.

2. Whenever I use this backend template, when I go to editing some templates, IF I click "overrides" or templates description, then I want to click "Editor" button on the left, to go back to the files tree... it doesn't work at all. So I have to close/cancel the whole thing...

3. There's an option to choose editors in the upper right... but it does no respond at all and on can't choose mirrorcode, what gives?

**Comment by sandmanmedia** [comment]
Yes, it's an incomplete project that I'm hoping people will want to contribute to the items you mentioned below.

Sent from my iPhone

On Jul 15, 2015, at 6:19 PM, jeegg <notifications@github.com<mailto:notifications@github.com>> wrote:


I love this theme but I have three issues.

  1.  When installing on Joomla! 3.4.3 I got an error that there's no en-GB.tpl_sandmanctrl.ini in the package, though it installed fine. I have tried adding another file with such name, the error was gone during install then.

  2.  Whenever I use this backend template, when I go to editing some templates, IF I click "overrides" or templates description, then I want to click "Editor" button on the left, to go back to the files tree... it doesn't work at all. So I have to close/cancel the whole thing...

  3.  There's an option to choose editors in the upper right... but it does no respond at all and on can't choose mirrorcode, what gives?

**Comment by jeegg** [comment]
I see. Sorry I can't be more of help then, since I'm pretty new to web dev. I do like this theme of yours though so *thumbs up* and let's hope someone can chime in with some solution to these issues.
